### PR TITLE
add pandarallel

### PR DESCRIPTION
**Pull request by hadim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hadim** [comment]
Generated using grayskull.

File: recipes/pandarallel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/nalepae/pandarallel
+  summary: An easy to use library to speed up computation (by parallelizing on multi CPUs) with pandas.
+  license: BSD-3-Clause
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/pandarallel/meta.yaml:40]: Add comment above `license_file`:

```
# license file manually packaged. See https://github.com/nalepae/pandarallel/pull/113
license_file: LICENSE
```

I've submitted a PR to include the license file in the source distribution (https://github.com/nalepae/pandarallel/pull/113). The comment will be a reminder to check the status of this on future releases of the package and updates to the feedstock.

**Review by synapticarbors — COMMENTED** [review]
I'm not sure if `noarch: python` is appropriate here since this is known not to work on windows (it only runs on the WSL). I'm not sure what the conda-forge policy on this is.

cc/ @conda-forge/staged-recipes

File: recipes/pandarallel/meta.yaml (hunk 1 above)
Comment (hadim) [reply recipes/pandarallel/meta.yaml:40]: Thank you for having a look but I tried that lib locally and it hangs forever when using it. I don't have time to debug this so I will just close this PR for the moment.

Comment (hadim) [reply recipes/pandarallel/meta.yaml:40]: (sorry I should have close it before...)

**Pull request closed without merging**